app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
While in this case we aren't required to do so, it's still good to get used to adding `dependent :destroy` into your `has_many` associations. For instance in this case we normally would want to remove all `user` 's posts and comments if the user's account is deleted

Reply:
Added the dependant :destroy to the has many assosiation.